src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -128,6 +132,20 @@ public void authorConfig_specialCharacter_success() throws IOException, InvalidL
         Assert.assertEquals(AUTHOR_CONFIG_SPECIAL_CHARACTER_AUTHORS, config.getAuthorList());
     }
 
+    @Test
+    public void authorConfig_multipleEmails_success() throws IOException, InvalidLocationException {
```

Comment:
similar test cases for ```singleEmail```, ```defaultEmail``` and ```noEmail```

Reply:
> similar test cases for singleEmail, defaultEmail and noEmail

This does not add any chances of catching new bugs, the reason I add this test is just to ensure that emails is a multiple field in author-config.csv.